src/files-rename.py

```diff
@@ -0,0 +1,24 @@
+import ctypes
+import os
+import sys
+
+AT_FDCWD = -100
+
+RENAME_NOREPLACE = (1 << 0)
+# RENAME_EXCHANGE = (1 << 1)
+
+libc6 = ctypes.CDLL(None, use_errno=True)
+
+if not libc6.renameat2:
```

Comment:
This seems unnecessary — it's been present since glibc 2.28.  If we didn't have this wrapper, the bridge wouldn't be running.

Reply:
Dropped.